Polyhedron/demo/Polyhedron/Plugins/Surface_mesh_deformation/Scene_edit_polyhedron_item.cpp

```diff
@@ -548,10 +581,16 @@ void Scene_edit_polyhedron_item_priv::remesh()
     deform_mesh->roi_vertices().begin(),deform_mesh->roi_vertices().end());
 
   ROI_faces_pmap roi_faces_pmap;
+  QVector<Point> controls_to_save;
```

Comment:
right. You should save `vertex_descriptor`'s.
Same remark for `points_to_save` below